src/opt.rs

```diff
@@ -223,6 +223,13 @@ pub struct MonitorConfig {
 	/// configured, it might re-try and lose funds through transaction fees/deposits.
 	#[clap(long, short, env = "SEED")]
 	pub seed_or_path: String,
+
+	/// Verify the submission by `dry-run` the extrinsic to check the validity.
+	/// If it fails then the block is ignored.
```

Comment:
What does this mean? What block is ignored? 

I guess what you are trying to say is: If this is set, against a node that does not support unsafe RPCs, then we don't submit anything? 

I think we should test this at startup, and immediately terminate. Every time we connect to a node, if we have this flag set, we should check it it supports the unsafe stuff or not.

Reply:
Nah, it means if the dry-run fails on a solution at block x then we simply emit a warning and drop that solution.
Then we try on the next block.

Fair enough, it could be good to have a proper error if the RPC call itself is denied because of "unsafe", currently we just terminate when that occurs with error from substrate.